File: ruby/codeeval/025_odd_numbers.rb
# def odds
#     (1..100).each do |i|
#         puts i if i % 2 == 1
#     end
# end
# odds

def odds
    last = []
    (1..50).each do |i|
        if i % 2 == 1
          puts i
          last.unshift 100 - i
        end
    end
    puts last.join("\n")
end
odds

File: javascript/codeeval/025_odd_numbers.js
function odds () {
	var last = []
	for (var i = 0; i < 50; i++) {
		if (i % 2 == 1) {
			console.log(i);
			last.unshift(100 - i);
		}
	}
	console.log(last.join("\n"));
}

odds();
